src/BizHawk.Client.EmuHawk/tools/TAStudio/FramesPrompt.Designer.cs

```diff
@@ -91,7 +91,7 @@ private void InitializeComponent()
 			this.Name = "FramesPrompt";
 			this.ShowIcon = false;
 			this.StartPosition = System.Windows.Forms.FormStartPosition.CenterParent;
-			this.Text = "Insert Frames";
+			this.Text = headMessage;

```

Comment:
Does this work in the designer?

Reply:
It won't, this assignment needs to be the empty string and another assignment can be done in the ctor after `InitializeComponent`.

Reply:
I've resolved this, thanks for the input